freqtrade/constants.py

```diff
@@ -484,6 +486,34 @@
             },
             'required': ['process_throttle_secs', 'allowed_risk']
         },
+        'external_message_consumer': {
+            'type': 'object',
+            'properties': {
+                'enabled': {'type': 'boolean', 'default': False},
+                'producers': {
+                    'type': 'array',
+                    'items': {
+                        'type': 'object',
+                        'properties': {
+                            'name': {'type': 'string'},
+                            'url': {'type': 'string', 'default': ''},
+                            'ws_token': {'type': 'string', 'default': ''},
+                        },
+                        'required': ['name', 'url', 'ws_token']
+                    }
+                },
+                'wait_timeout': {'type': 'integer'},
+                'sleep_time': {'type': 'integer'},
+                'ping_timeout': {'type': 'integer'},
```

Comment:
should define a `"minimum"` of 0 i think - i don't think negative ints will be supported where they're used.